src/ProjectTemplates/test/GrpcTemplateTest.cs

```diff
@@ -33,7 +33,7 @@ public ITestOutputHelper Output
         }
 
         [ConditionalFact]
-        [SkipOnHelix("Not supported queues", Queues = "Windows.7.Amd64;Windows.7.Amd64.Open;Windows.81.Amd64.Open;All.OSX")]
+        [SkipOnHelix("Not supported queues", Queues = "Windows.7.Amd64;Windows.7.Amd64.Open;Windows.81.Amd64.Open;All.OSX;(Alpine.312.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200908125345-56c6673")]
```

Comment:
Should we future proof this now and introduce/generalize the alpine to be similar to All.OSX skip?

Reply:
i.e. more cases or more smarts to https://github.com/dotnet/aspnetcore/blob/master/src/Testing/src/xunit/SkipOnHelixAttribute.cs#L54 and potentially https://github.com/dotnet/aspnetcore/blob/master/eng/targets/Helix.targets#L100 for All support in SkipHelixQueues